docs/DeveloperGuide.md

```diff
@@ -192,11 +192,19 @@ contains a `toString` method that overrides the format of math module printing.
 #### 3.4.2 `list` Package
 
 ### 3.5 Storage Component
-<b>API</b>: `Storage.java`
+Class Diagram:
+
+![archi](./images/storageClassDiagram.jpg)
```

Comment:
Why don't you consider maintaining a unified style for class diagram? Perhaps different color for each component will allow the reader to better understand the class diagrams